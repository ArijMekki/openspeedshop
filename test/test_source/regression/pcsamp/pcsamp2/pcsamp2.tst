echo expcreate -f  ../../../../executables/mutatee/cplus_version/mutatee pcsamp >> input.script
echo expgo >> input.script
echo expview vtop5 >>input.script
openss -batch < input.script
