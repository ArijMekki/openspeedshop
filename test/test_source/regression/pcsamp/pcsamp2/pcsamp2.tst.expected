  CPU Time (Seconds)  Function
           3.000000  f2
           2.0000  f3
            1.0000  f1
